decidim/decidim
# Fix blocked users not present in global moderation panel

#### :tophat: What? Why?
When I am blocking an user without any kind of moderation, that particular user is being blocked, yet the user will not be visible in the Blocked user panel.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #10521 

#### Testing

1. Go to Participants Admin panel
2. Block any users
3. Visit the global moderation panel (blocked user section)
4. See the user is not present.
5. Apply patch, run rake task, 
6. See the old blocked users are present 
7. Repeat  2,3,4
8. See behavior changed

:hearts: Thank you!
